### PR TITLE
version bump, changelog update

### LINKED ISSUE #4: Consume multiple locales
Issue: how to integrate node packages that have their own translations?

Proposed solution: Copy package files into a build and update this plugin to consume translation catalogs from multiple sources.

Assuming we have `app` and installable `foo`:
1. copy `app` source files to `build`.
2. copy `foo` files to `build`.
3. configure `gulp-static-i18n` to consume translation catalogs from `app`'s locale and `foo`'s locale.
4. `build` is then statically translated.

ping @euwest

### PR DESCRIPTION
**Pull request by euwest**
closes https://github.com/yola/gulp-static-i18n/issues/4

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # gulp-static-i18n changelog
 
+## 0.0.2
+
+* Allow sourcing of PO files from Multiple directories
```
Comment (beck) [inline CHANGELOG.md]: changelogs are a good place to note breaking changes. Note that the config has changed from `localeDir` to `localeDirs`.  This change also needs to be reflected in the readme.

Comment (euwest) [inline CHANGELOG.md]: oh right, forgot about that. in that case this should be `1.0.0`, shouldn't it? since it's a guaranteed breaking change?

Comment (beck) [inline CHANGELOG.md]: No. `1.x.x` means that we have a stable release.
We will keep it `0.0.x` to indicate that this code is in flux.
When it gets a bit more mellow we'll bump the minor.
When it's lookin solid, then we'll bump major.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
 # gulp-static-i18n changelog
 
+## 0.0.2
+
+* Allow sourcing of PO files from Multiple directories.
+Change `localeDir` to `localeDirs` in options object passed
```
Comment (beck) [inline CHANGELOG.md]: Use two bullets

File: README.md [hunk 3]
```diff
@@ -39,7 +39,7 @@ module.exports = gulp;
 
 ### What happens on disk
 
-The source is copied for each language present in the locale directory. Every
+The source is copied for each language present in each locale directory. Every
```
Comment (beck) [inline README.md]: Not true, the first locale is the one that were the langs

File: README.md [hunk 4]
```diff
@@ -37,10 +37,10 @@ gulp.task('translate', ['build'], function(){
 module.exports = gulp;
 ```
 
-### What happens on disk
+### What happens:
```
Comment (beck) [inline README.md]: Unless content relevant, titles to do get line-ending punctuation.

But this title is now a bit usless.  maybe it should be: `Creating language builds`

File: README.md [hunk 5]
```diff
@@ -37,10 +37,10 @@ gulp.task('translate', ['build'], function(){
...
 
-The source is copied for each language present in the locale directory. Every
-file is pruned for translation calls.
+`po` files are catalogued/merged from each locale directory, then used
```
Comment (beck) [inline README.md]: Merging of po files is standard behavior when defining multiple catalogs.  No need to say that here.

The purpose of this section is to document how the plugin is unique, specifically that it creates an individual build for each target language, translates on disk, and removes the translation infrastructure from the source.  (maybe it should be said kinda like so)

**Comment by beck** [comment]
:+1:

**Pull request merged**